Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM python:3
+
+# install conda
+RUN curl -sSLO https://repo.continuum.io/miniconda/Miniconda3-4.5.12-Linux-x86_64.sh && \
```

Comment:
is there a benefit to installing miniconda in a python3 image?

Reply:
I wanted to be explicit in the conda version, but it's clearly redundant. Can remove